### PR TITLE
Add example for apptainer running on rocky

### PR DESCRIPTION
**Pull request by afbjorklund**
The new version is not yet available for fedora or centos,
but it will be available from EPEL after the 1.1.0 release.

https://github.com/apptainer/apptainer/releases/tag/v1.0.2

### CODE REVIEW

File: examples/apptainer.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  system: false
+  user: false
+provision:
+- mode: system
+  script: |
+    #!/bin/bash
+    set -eux -o pipefail
+    command -v apptainer >/dev/null 2>&1 && exit 0
+    url="https://github.com/apptainer/apptainer"
+    tag="$(basename $(curl -fs -o/dev/null -w %{redirect_url} $url/releases/latest))"
```
Comment (AkihiroSuda) [inline examples/apptainer.yaml:35]: Can we pin a specific version?

File: examples/README.md [hunk 2]
```diff
@@ -20,6 +20,7 @@ Container engines:
 - [`docker.yaml`](./docker.yaml): Docker
 - [`podman.yaml`](./podman.yaml): Podman
 - [`singularity.yaml`](./singularity.yaml): Singularity
```
Comment (AkihiroSuda) [inline examples/README.md:22]: Can we now remove singularity? Or maybe move to the `deprecated` directory

File: examples/apptainer.yaml (hunk 1 above)
Comment (afbjorklund) [reply examples/apptainer.yaml:35]: Sure, current is `1.0.2`

File: examples/README.md (hunk 2 above)
Comment (afbjorklund) [reply examples/README.md:22]: This new "apptainer" package is not yet available in any distributions, though.

Comment (afbjorklund) [reply examples/README.md:22]: Requested a package for aarch64, currently the _only_ architecture is x86_64

Comment (afbjorklund) [reply examples/README.md]: I moved it to `experimental`, and then removed the arm64 arch from list (for now)

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**